ga4gh/backend.py

```diff
@@ -811,10 +927,12 @@ class FileSystemBackend(AbstractBackend):
     def __init__(self, dataDir):
         super(FileSystemBackend, self).__init__()
         self._dataDir = dataDir
-        sourceDirNames = ["referenceSets", "datasets"]
+        sourceDirNames = ["referenceSets", "ontologies", "datasets"]
         constructors = [
-            references.HtslibReferenceSet, datasets.FileSystemDataset]
-        objectAdders = [self.addReferenceSet, self.addDataset]
+            references.HtslibReferenceSet, datasets.FileSystemOntology,
```

Comment:
So, if FileSystem ontology maps to a single Ontology, and each ontology is a single txt file in the 'ontologies' directory, this will all work out nicely.